includes/Core/Email_Reporting/Email_Report_Report_Processor.php

```diff
@@ -0,0 +1,361 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Email_Reporting\Email_Report_Report_Processor
+ *
+ * @package   Google\Site_Kit\Core\Email_Reporting
+ * @copyright 2025 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Email_Reporting;
+
+/**
+ * Helper class to normalize and process report payloads for email sections.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Email_Report_Report_Processor {
+
+	/**
+	 * Processes GA4 batch reports using helper structure.
+	 *
+	 * @param array $batch_results  Raw batch report results.
+	 * @param array $report_configs Optional. Additional report config metadata keyed by index.
+	 * @return array Processed reports keyed by report identifier.
+	 */
```

Comment:
Missing `@since` tag

```suggestion
	/**
	 * Processes GA4 batch reports using helper structure.
	 *
	 * @since n.e.x.t
	 *
	 * @param array $batch_results  Raw batch report results.
	 * @param array $report_configs Optional. Additional report config metadata keyed by index.
	 * @return array Processed reports keyed by report identifier.
	 */
```

Also for the rest of the methods in the class